src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto font', sans-serif;
+}
+
+.heading {
+  text-align: center;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  background-color: #f8f8f8;
+}
+
+.logo {
+  margin: 0 40px;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+}
+
+nav {
+  flex-grow: 1;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__list li {
+  margin: 0 15px;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0 60px;
+  padding: 0;
+  justify-content: flex-end;
+  display: flex;
+  font-size: 14px;
+  font-weight: bold;
+  align-items: center;
+}
+
+.nav__link {
+  width: 1200px;
+  height: 60px;
+  left: -600px;
```

Comment:
The use of fixed width, height, and left properties for '.nav__link' can lead to layout issues on different screen sizes. Consider using relative units or flexbox properties to ensure better responsiveness.